requirements.txt

```diff
@@ -17,3 +17,4 @@ geopandas
 setuptools
 pyzmq==25.0.1
 cluster @ git+https://code.it4i.cz/def/cluster.git#e47b756e1bcb91c0244e1cede58c868d4ade8e6f
+h5py
```

Comment:
I know that it's not done for most existing dependencies, but at least for new ones, please add a version requirement, like `h5py==1.2.3`. To show the version that you are currently using, you can run `pip freeze | grep h5py` (or something similar on Windows :) ).